### ISSUE TITLE
failed to decrypt

### ISSUE THREAD

**Issue by muthusankar54** [ulisesbocchio/jasypt-spring-boot#149]
Hi UB/Team,
 
         Currently, am using the version of Jaspyt 2.1.0  it's failed to decrypt the encrypted which I was confgures in the application.properties, and for the simple Encryption purpose I used PooledPBEStringEncryptor and returning the StringEncryptor object. when it's tried to decrypt in  @Value("${test.key}") in service, it's failed.

and I am not able to find the method **setIvGeneratorClassName**  in PooledPBEStringEncryptor?

can anyone help to resolve this?

```
	@Bean("jasyptStringEncryptor")
	public StringEncryptor stringEncryptor() {
		PooledPBEStringEncryptor encryptor = new PooledPBEStringEncryptor();
		SimpleStringPBEConfig config = new SimpleStringPBEConfig();
		config.setPassword("test");
		config.setAlgorithm("PBEWithMD5AndDES");
		config.setKeyObtentionIterations("1000");
		config.setPoolSize("1");
		config.setProviderName("SunJCE");
		config.setSaltGeneratorClassName("org.jasypt.salt.RandomSaltGenerator");
**@ulisesbocchio  can you check the below line:**
		**//config.setIvGeneratorClassName("org.jasypt.salt.NoOpIVGenerator");**
		config.setStringOutputType("base64");
		encryptor.setConfig(config);
		return encryptor;
	}
```

**Comment by melloware** [comment]
This property 
```java
config.setIvGeneratorClassName("org.jasypt.salt.NoOpIVGenerator");
```

Is only available in 2.1.1 not 2.1.0 which you are using.

**Comment by ulisesbocchio** [comment]
thanks @melloware
